src/modules/navigator/navigator_main.cpp

```diff
@@ -623,15 +630,18 @@ int navigator_main(int argc, char *argv[])
 
 	} else if (!strcmp(argv[1], "status")) {
 		navigator::g_navigator->status();
+		return 0;
```

Comment:
These "return 0" calls are not needed for every else. You will go to the bottom and do a return 0.